lib/tasks/load_tags_from_content_api.rake

```diff
@@ -0,0 +1,39 @@
+desc "Load tags from Content API into Collections Publisher."
+task :load_tags_from_content_api => :environment do
+  require 'gds_api/content_api'
+
+  content_api_endpoint = Plek.new.find("content_api")
+  content_api = GdsApi::ContentApi.new(content_api_endpoint)
+
+  puts "There are currently #{Tag.count} tags in the database"
+  puts "Fetching all tags from the Content API..."
+  tags = content_api.get_list!("#{content_api_endpoint}/tags.json")
+
+  puts "Received a total of #{tags.count} tags"
+  puts "Loading tags into Collections Publisher..."
+  tags.each do |api_tag|
+    unless Tag.where(slug: api_tag.slug).any?
+      tag = Tag.new(slug: api_tag.slug,
+                    description: api_tag.details.description,
+                    title: api_tag.title)
```

Comment:
We ought to be creating tags of the approprite type (`Topic` or `MainstreamBrowsePage`) here.  As I understand it, there shouldn't really be any instances of the base `Tag` class in the db.